arcade/draw_commands.py

```diff
@@ -632,7 +640,36 @@ def draw_lrtb_rectangle_outline(left: float, right: float, top: float,
     :Raises AttributeError: Raised if left > right or top < bottom.
 
     """
+    if left > right:
+        raise AttributeError("Left coordinate must be less than or equal to "
+                             "the right coordinate")
+
+    if bottom > top:
+        raise AttributeError("Bottom coordinate must be less than or equal to "
+                             "the top coordinate")
 
+    center_x = (left + right) / 2
+    center_y = (top + bottom) / 2
+    width = right - left
+    height = top - bottom
+    draw_rectangle_outline(center_x, center_y, width, height, color,
+                           border_width)
+
+
+def draw_lrbt_rectangle_outline(left: float, right: float, bottom: float, top: float, color: Color,
+                                border_width: float = 1):
+    """
+    Draw a rectangle by specifying left, right, bottom and top edges.
+
+    :param float left: The x coordinate of the left edge of the rectangle.
+    :param float right: The x coordinate of the right edge of the rectangle.
+    :param float bottom: The y coordinate of the rectangle bottom.
+    :param float top: The y coordinate of the top of the rectangle.
+    :param Color color: The color of the rectangle.
+    :param float border_width: The width of the border in pixels. Defaults to one.
+    :Raises AttributeError: Raised if left > right or top < bottom.
```

Comment:
`s/Attribute/Value/`

Reply:
Updated in https://github.com/pythonarcade/arcade/pull/1650/commits/0af3ba990f9eacd0a13e188aa300615db4226460